client/connect-account-page/strings.tsx

```diff
@@ -111,19 +116,27 @@ export default {
 				'What else should I keep in mind while completing this process?',
 				'woocommerce-payments'
 			),
-			description: __(
-				"If you're setting up Woo Payments for someone else, it's best to have that person complete the account creation process. As you can see above, we ask for very specific information about the business owner - and you might not have all the details at hand. It's not always possible to change account information once it's been saved, especially if the site accepts live transactions before the correct account information is entered.",
-				'woocommerce-payments'
+			description: sprintf(
+				/* translators: %s: WooPayments */
+				__(
+					"If you're setting up %s for someone else, it's best to have that person complete the account creation process. As you can see above, we ask for very specific information about the business owner - and you might not have all the details at hand. It's not always possible to change account information once it's been saved, especially if the site accepts live transactions before the correct account information is entered.",
+					'woocommerce-payments'
+				),
+				'WooPayments'
 			),
 		},
 		isMyDataSafe: {
 			heading: __(
 				'Is my data safe with WooCommerce?',
```

Comment:
I think this string is wrong. It probably should be "Is my data safe with WooPayments?"

Reply:
Good point! Fixed in https://github.com/Automattic/woocommerce-payments/pull/6766/commits/da8f78444370e5659a110af90b669c3379929721